registry/client/src/authEntities/dataTransform.js

```diff
@@ -0,0 +1,11 @@
+export function transformGet(entity) {
+
+}
+
+export function transformSet(entity, operation) {
+    if (operation === 'update') {
```

Comment:
In my opinion, it would be nice to have one object with operations and use it everywhere
because as I see operations are copy/pasted in each of the files

Reply:
fixed